src/dream.mli

```diff
@@ -1583,15 +1583,15 @@ val invalidate_session : request -> unit promise
 
 (** {2 Back ends} *)
 
-val memory_sessions : ?lifetime:float -> middleware
+val memory_sessions : middleware
```

Comment:
What happened to the `?lifetime` parameter?

Reply:
I was forced to eliminate it for saving existing way of usage session middlewares. In other way every usage of session middleware would look like `Dream.memory_sessions ()`.

Reply:
I think this parameter has to be kept, and, in light of the other comment, can be kept.